src/connections/Entity.ts

```diff
@@ -28,9 +30,9 @@ import { connectedResourceDeleter, resourcesDeletedSelector } from "@/utils/conn
 import { connectionHook, connectionLoader } from "@/utils/connectionShortcuts";
 import { selectorCache } from "@/utils/selectorCache";
 
-export type EntityFullDto = ProjectFullDto | SiteFullDto | NurseryFullDto;
-export type EntityLightDto = ProjectLightDto | SiteLightDto | NurseryLightDto;
-export type EntityDtoType = EntityFullDto | EntityLightDto | NurseryLightDto;
+export type EntityFullDto = ProjectFullDto | SiteFullDto | NurseryFullDto | ProjectReportFullDto;
+export type EntityLightDto = ProjectLightDto | SiteLightDto | NurseryLightDto | ProjectReportLightDto;
+export type EntityDtoType = EntityFullDto | EntityLightDto | NurseryLightDto | ProjectReportLightDto;
```

Comment:
This should not need `NurseryLightDto` or `ProjectReportLightDto` - they're already included in the union of `EntityFullDto | EntityLightDto`